cmd/buck/cli/cli.go

```diff
@@ -94,6 +95,15 @@ func Init(baseCmd *cobra.Command) {
 	rolesGrantCmd.Flags().StringP("role", "r", "", "Access role: none, reader, writer, admin")
 
 	linksCmd.Flags().String("format", "default", "Display URL links in the provided format. Options: [default,json]")
+
+	remoteCmd.AddCommand(remoteAddCmd)
+	remoteCmd.AddCommand(remoteRemoveCmd)
+
+	remoteAddCmd.Flags().BoolP("yes", "y", false, "Skips the confirmation prompt if true")
+	remoteAddCmd.Flags().BoolP("quiet", "q", false, "Write minimal output")
+
+	remoteRemoveCmd.Flags().BoolP("yes", "y", false, "Skips the confirmation prompt if true")
+	remoteRemoveCmd.Flags().BoolP("quiet", "q", false, "Write minimal output")
```

Comment:
New `remote add` and `remove rm` commands that function like the original `buck` command: ignore the local repo. This is useful for very large bucket or buckets with many small files where calculating local diffs takes a very long time.

Reply:
oh this is really nice!